docs/rules/rrd/huge-files.md

```diff
@@ -0,0 +1,23 @@
+# Huge Files
+
+Checks if the combined length of script, template, and CSS in a Vue component is too long. The default threshold for a warning is 300 lines, and for an error is 500 lines.
```

Comment:
In other rules error is the double of the warning. Can we do it here also? Or did you have some reason for these numbers?

Reply:
Oh the numbers were provided by you in the issue description 👀 but you are right, let's keep using the 2 * WARNING TRESHOLD

Reply:
fixed in last commit